hosts/signed-int16/home/reisen/home.nix

```diff
@@ -7,15 +7,60 @@
   ...
 }: {
   imports =
-    [
+    [ 
```

Comment:
Before commiting, there's a pre-commit configured for formatting a code, if pre-commit fails just run manually nix fmt, commit in nix develop mode
```suggestion
    [
```

Reply:
let's ignore the fact that the first commit was made without nix develop mode